### PR TITLE
Dev/kduran/vbs22 63 creacion viaje

### PR DESCRIPTION
**Pull request by kendallduranv**
Form linked to food component, the version with steroids came with the branch of "Baarra de seguimiento" which contains the progress budget bar and also linked the lodge card

### CODE REVIEW

**Review by DieS06 — APPROVED** [review]

**Comment by DieS06** [comment]
Resolve conflicst pending!
src/app/components/lodge/lodge-card/lodge-card.component.html
src/app/components/maps/maps.component.html
src/app/pages/lodge/lodge.component.ts

**Review by DieS06 — APPROVED** [review]

File: src/app/components/map/map.component.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <!-- Add the required attributes to the google-map component -->
-<google-map class="maps" [options]="options"></google-map>
+<google-map class="maps" height="100vh" width="100wv"   [options]="options"></google-map>
```
Comment (nefi-cenfo) [inline src/app/components/map/map.component.html:2]: Be aware of the misspelling of the unit 'vw'.

File: src/app/pages/trip-form/trip-form.component.scss [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+.inputs-card {
+    width: 100%;
+    max-height: 100%;
+    background: $background-color-white;
+    border-radius: 6px;
+    border-width: 1px;
+    border-color: $image-backgorund-color;
+    border-style: solid;
+    // background-color: $background-color-white;
```
Comment (nefi-cenfo) [inline src/app/pages/trip-form/trip-form.component.scss:59]: Please remove the line commented.

File: src/app/pages/trip-form/trip-form.component.ts [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }
+  
+
+  setTripInfo(formGeneralInfo: any, formFlightInfo: any,event:Event){
+    event.preventDefault();
+    ;
+    if (formGeneralInfo.valid && formFlightInfo.valid) {
+
+      this.tripService.setInitialForm(this.tripFormNgModel);
+      // this.router.navigateByUrl('/lodge')
```
Comment (nefi-cenfo) [inline src/app/pages/trip-form/trip-form.component.ts:57]: Please delete the line commented.

File: src/app/services/trip.service.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+
+  getAllSignal() {
+    this.findAll().subscribe({
+      next: (response: any) => {
+        response.reverse();
+        this.tripFormSignal.set(response);
+      },
+      error: (error: any) => {
+        console.error('Error fetching users', error);
```
Comment (nefi-cenfo) [inline src/app/services/trip.service.ts:62]: Please change 'users' for 'trips'.

**Review by nefi-cenfo — APPROVED** [review]

**Pull request merged**